scripts/demos/common/databases.sh

```
@@ -0,0 +1,141 @@
+  mkdir -p "$destination"
+  rsync -a --delete "$source"/ "$destination"/
+  cp "$source_marker" "$destination_marker"
+}
+
+mark_database_dir_dirty() {
+  local directory="$1"
+  rm -f "$directory/.$DEMO_NAME-source.json"
+}
+
+refresh_from_mithril() {
+  cd "$AMARU_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)